app/assets/stylesheets/organizations.scss

```diff
@@ -2,6 +2,47 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
+.header-container {
+  width: 100%;
+  min-height: 58vw;
+  border-left: none;
+  border-right: none;
+  position: relative;
+  padding: 20px;
+  position: inherit;
+
+  .video-container {
+  position: absolute;
+  top: 0%;
+  left: 0%;
+  height: 100%;
+  width: 101%;
+  overflow: hidden;
+  margin-left: -1%;
+}
+  video {
+  position: absolute;
+  zoom: 110%;
+  z-index: -1;
+  width: 100%;
+  opacity:0.78;
+  }
+
+  h2{
+    opacity: 0.60;
+    font-size: 5vw;
+    margin-top: 10%;
+    width: 100%;
+    text-align: center;
+  }
+  p{
+    font-size: 2vw;
+    width: 100%;
+    color: #0E426C;
+    text-align: center;
+  }
+}
```

Comment:
Make sure to use classes or ids to style on this page, only referencing `h2` or `p` will change them globally in the project